stable/centrifugo/values.yaml

```diff
@@ -0,0 +1,48 @@
+replicaCount: 1
+image:
+  repository: centrifugo/centrifugo
+  tag: 1.7.3
+  pullPolicy: IfNotPresent
+service:
+  name: http
+  type: ClusterIP
+  externalPort: 8000
+  internalPort: 8000
+ingress:
+  enabled: true
```

Comment:
I would not enable ingress by default. The configuration assume there is an Nginx ingress controller. I would not require this by default.